### PR TITLE
Update to JDK 11 and sbt 1.3.8

### PR DESCRIPTION
**Pull request by lukestephenson-zendesk**
About to run this training and everyone here is now on jdk 11.
Bumping the version so people don't need to install an older version.

Feel free to ignore though if you aren't ready to bump to jdk 11.

### CODE REVIEW

**Comment by wjlow** [comment]
@lukestephenson-zendesk can you let me know here once you've run it on JDK 11 without any issues?

**Comment by lukestephenson-zendesk** [comment]
@wjlow Ran the training on JDK11.  No issues.

File: auto/dev-environment [hunk 1]
```diff
@@ -7,10 +7,11 @@ trap "docker-compose down --volumes --remove-orphans" 0
 
 # Create volume
 docker volume create --name ivy-cache > /dev/null
+docker volume create --name coursier-cache > /dev/null
 docker volume create --name docker-scala-sbt-cache > /dev/null
```
Comment (felipeeflores) [inline auto/dev-environment]: given that this change upgrades sbt and starts using coursier the `docker-scala-sbt-cache` becomes unused, so probably should be removed as part of this PR.

Comment (lukestephenson-zendesk) [reply auto/dev-environment]: Removed.  Thanks @felipeeflores

**Pull request merged**